tests/unit/backend/controllers/email-submissions.server.controller.spec.js

```diff
@@ -2527,6 +2527,7 @@ describe('Email Submissions Controller', () => {
                 value: 2,
               },
             ],
+            logicType: 'showFields',
```

Comment:
Why can the tests pass last time?

Reply:
Ahaha I think previously since there was no logic type, all the elements of `form_logics` would just get filtered out and the logic module would think that there was no logic...oops. So then when I added `isShowFieldsLogic`, it returned false when `logicType` was undefined, so the logic module interpreted it as `preventSubmit` so the tests broke.